test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4495,22 +4512,7 @@ def _start(self, inst=None, args=None, cmd_map=None, launcher=None):
         if ret['err']:
             ret_msg = ret['err']
 
-        self.pid = self._get_pid(inst)
-        # get_pid gets information from a lock file that may not have been
-        # removed when the daemon stopped so we verify that the PID is
-        # actually alive in the list of pids returned by ps
-        live_pids = self._all_instance_pids(inst)
-        i = 0
-        while ((self.pid is None) or
-               (live_pids is None or self.pid not in live_pids)) and (i < 30):
-            time.sleep(1)
-            i += 1
-            live_pids = self._all_instance_pids(inst)
-            self.pid = self._get_pid(inst)
-            if live_pids is not None and self.pid in live_pids:
-                return ret_msg
-        if i == 30:
-            raise PbsServiceError(rv=False, rc=-1, msg="Could not find PID")
+        self._update_pid(inst)
```

Comment:
Oops... Thanks for catching this. It should raise PbsServiceError in case of pid=None. changed accordingly.